IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -449,6 +468,19 @@ - (BOOL)validateAndRemoveRefreshToken:(MSIDRefreshToken *)token
                               context:(id<MSIDRequestContext>)context
                                 error:(NSError **)error
 {
+    return [self validateAndRemoveRefreshableToken:token
```

Comment:
why an abstraction here with simply passing along every input param coming into next and does nothing else?

Reply:
Now I added `validateAndRemovePrimaryRefreshToken` which will use the above abstraction :)